app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes, :comments
+
+  after_save :update_user_post_counter
+
+  def return_most_recent_comments
+    comments.last(5)
+  end
+
+  private
+
+  def update_user_post_counter
+    user.posts_counter += 1
```

Comment:
- [ ] <b>REQUIRED:</b> Now that you have changed the object `user` to `author` as I have suggested above. you can increment counter like so:

```ruby
 author.increment!(:posts_counter)

Reply:
This change has been done well